src/main/java/seedu/address/model/common/Name.java

```diff
@@ -16,6 +16,18 @@ protected Name(String name) {
         this.fullName = name;
     }
 
+    /**
+     * Returns true if given {@code substring} is part of this {@code Name}.
+     * This method is case-insensitive.
+     *
+     * @param substring Substring to match this Name against.
+     * @return True if given substring is part of this Name.
+     */
+    public Boolean containsIgnoreCase(String substring) {
```

Comment:
just a nitpick, is there a reason why `Boolean` is used instead of `boolean`? 
Also good job in abstracting this method out since it would be used multiple times for both `findevent` and also `findfriend`!

Reply:
Typo, will fix